nix/cassandra.nix

```diff
@@ -0,0 +1,169 @@
+# Based on https://github.com/cachix/devenv/blob/fa9a708e240c6174f9fc4c6eefbc6a89ce01c350/src/modules/services/cassandra.nix
+{ pkgs, lib, name, config, ... }:
+let
+  inherit (lib) types;
+  yamlFormat = pkgs.formats.yaml { };
+in
+{
+  options = {
+    enable = lib.mkEnableOption name;
+
+    package = lib.mkPackageOption pkgs "cassandra" { };
+
+    dataDir = lib.mkOption {
+      type = types.str;
+      default = "./data/${name}";
+      description = "The cassandra data directory";
+    };
+
+    listenAddress = lib.mkOption {
+      type = types.str;
+      description = "Listen address";
+      default = "127.0.0.1";
+      example = "127.0.0.1";
+    };
+
+    nativeTransportPort = lib.mkOption {
```

Comment:
Perhaps the `Getting started` can also mention how to change the port because it is not straightforward.